### PR TITLE
AndroidProject.setPackageName: update build.gradle applicationId

### PR DESCRIPTION
**Pull request by ksteuer**
Update build.gradle applicationId when calling project.setPackageName. #24

### CODE REVIEW

File: packages/project/src/android/project.ts [hunk 1]
```diff
@@ -50,7 +50,10 @@ export class AndroidProject {
   }
 
   async setPackageName(packageName: string) {
-    this.manifest.getDocumentElement()?.setAttribute('package', packageName);
+    await Promise.all([
+      this.manifest.getDocumentElement()?.setAttribute('package', packageName),
```
Comment (mlynch) [inline packages/project/src/android/project.ts:54]: No need for the `Promise.all` here as the `setAttribute` is synchronous

**Review by mlynch — CHANGES_REQUESTED** [review]
Looks great, I wasn't expecting a PR and this is pretty much exactly what I had locally. Would love to have your contribution instead. Left one small comment

File: packages/project/src/android/project.ts (hunk 1 above)
Comment (ksteuer) [reply packages/project/src/android/project.ts:54]: Right! Changing it right away. Thanks!

**Comment by mlynch** [comment]
💪 awesome! Much appreciated

**Pull request merged**